.devcontainer/Dockerfile

```diff
@@ -68,3 +68,9 @@ RUN wget -O /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/
   && chmod +x /usr/local/bin/yq
 
 USER $USERNAME
+
+# Save command line history 
+RUN echo "export HISTFILE=$HOME/commandhistory/.bash_history" >> "$HOME/.bashrc" \
+    && echo "export PROMPT_COMMAND='history -a'" >> "$HOME/.bashrc" \
```

Comment:
Could this overwrite other bash history we may have from other devcontainers - do we need "tre" in there?

Reply:
Oh sorry - that is sorted out in the devcontainer.json - ignore me!

Reply:
The mounted volume has `tre` in it, so would only conflict if another dev container uses the same volume name :-)

```
		"type=volume,source=tre-bashhistory,target=/home/vscode/commandhistory",
```